pyClient/zeth/utils.py

```diff
@@ -232,8 +233,10 @@ def encode_to_hash(message_list: Any) -> bytes:
         m_hex = m
 
         # Convert it into a hex
-        if isinstance(m, int):
-            m_hex = "{0:0>4X}".format(m)
+        if isinstance(m, (FQ, int)):
+            assert int(m) < 2**256, "Encode input as bytes32 longer than 256 bits"
```

Comment:
`Encode input as bytes32 longer than 256 bits` -> `The input cannot be encoded as a bytes32 (longer than 256 bits)`

Reply:
Corrected